docs/en/engines/table-engines/mergetree-family/annindexes.md

```diff
@@ -142,6 +142,8 @@ was specified for ANN indexes, the default value is 100 million.
 
 - [Annoy](/docs/en/engines/table-engines/mergetree-family/annindexes.md#annoy-annoy)
 
+- [USearch](/docs/en/engines/table-engines/mergetree-family/annindexes.md#usearch-usearch)
```

Comment:
@rschu1ze, the logic seems to be preserved here. Annoy means "Approximate Nearest Neighbors Oh Yeah" while implementing the KD-Trees algorithm. In our case, we implement a variation of HNSW, but the library is called USearch. Moreover, keeping the USearch name makes it easier for people to infer the format of the proximity graph file, to be able to introspect the indexes.

Reply:
I had a second look, and while Annoy as a library indeed implements a variant of kd-trees, "Annoy" is also an established ANN algorithm name, e.g. [here](https://ann-benchmarks.com/index.html#algorithms). Likewise (same source), "hnsw" is an algorithm being implemented by different libraries.

But I don't want to split hairs here. I am good with calling the index type "usearch" BUT 1. the documentation should clearly state that this is a library that implements HNSW and 2. usearch should ideally not silently switch to another algorithm between releases. In that case, it may break results of existing queries and lead to different query runtimes.